cirq-google/cirq_google/ops/coupler_pulse.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An instance of FSimGate that works naturally on Google's Sycamore chip"""
+
+from typing import Optional, Tuple, TYPE_CHECKING
+
+import numpy as np
+
+from cirq import ops, protocols, value
```

Comment:
nit: can just do `import cirq` since it's a separate package.

Reply:
Done.